CONTRIBUTING.md

```diff
@@ -101,3 +101,9 @@ repository, we have a [community membership guide](./docs/governance/community_m
 
 - [How to write a good Git Commit message](https://chris.beams.io/posts/git-commit/) -
   Great way to make sure your Pull Requests get accepted.
+- **Log levels usage:**
+  - Fatal - a ctirical error has happened and application can not perform subsequent work any more. Examples: missing configuraion information in case there are no default values provided, one of the services can not start normally, etc.
```

Comment:
```suggestion
  - Fatal - a critical error has happened and application cannot perform subsequent work any more. Examples: missing configuration information in case there are no default values provided, one of the services can not start normally, etc.
```